src/common/containers/landing.js

```diff
@@ -42,9 +42,16 @@ class Landing extends Component {
               </ul>
 
               <div className={ styles.bannerButton }>
-                <Anchor href="/auth/authenticate" icon={ octocat } flavour='embiggened' appearance='positive' >
-                  Set up in minutes
-                </Anchor>
+                { !this.props.auth.authenticated &&
```

Comment:
Wouldn't that be a bit more clear as `{ this.props.auth.authenticated ? ... : ... }`? 

I as one statement it will be more close to `if ... else` than it is now as 2 separate conditions.